guide/preferences.md

```diff
@@ -40,6 +40,12 @@ depending on the GPU OpenGL abilities.
 the Camera device images. The methods are sorted from the most efficient one to
 the least efficient one.
 
+- The `Main 3D View Anti-Aliasing` option allows you to enable Multisample Anti-Aliasing
+on the 3D Scene in Webots. This option can lead to reduced performance, but it improves 
+graphical fidelity significantly. It is disabled by default on systems that do not meet 
+our minimum requirements. Note that this option does not apply to any Camera rendering, 
+this is managed by a separate setting in the same tab of the preferences dialog.
```

Comment:
I would explicitly make reference to this separate setting, by writing the exact option name, so that it is clear which option is referring to.